src/lib/components/multiDropzone/index.test.tsx

```diff
@@ -128,14 +128,14 @@ describe('MultiDropzone component', () => {
       const screen = setup({ accept: 'image' });
 
       expect(
-        screen.getByText('Supports HEIC, BMP, JPEG, JPG, PNG')
+        screen.getByText('Supports images, videos and documents')
```

Comment:
Shouldn't this show only "images" because we are explicitly adding the `{ accept: 'image' }` object?

It seems like we are dropping the labelling based on the file types. Is that what we want?

Reply:
@talo242 Fixed it. For custom accept types you need to provide your own label but for generic image/video/documents it will now show the default label.